bsoid_app/data_preprocess.py

```diff
@@ -1,225 +1,176 @@
+
 import os
 from datetime import date
-
+import sys
 import h5py
 import joblib
 import randfacts
 import streamlit as st
-
+                    
 from bsoid_app.bsoid_utilities import visuals
 from bsoid_app.bsoid_utilities.likelihoodprocessing import *
 from bsoid_app.bsoid_utilities.load_json import *
-
-
+                    
+working_dir = os.environ.get('WORKING_DIR_BSOID', '')
```

Comment:
Ah you can adjust the class `__init__` to specify these as inputs params (like on line 19), and in that case you don't need to read these here.